pkg/scaffold/olm-catalog/csv_test.go

```diff
@@ -51,24 +49,29 @@ func TestCSVNew(t *testing.T) {
 	csvVer := "0.1.0"
 	projectName := "app-operator"
 
-	err := s.Execute(&input.Config{ProjectName: projectName},
-		&CSV{CSVVersion: csvVer, pathPrefix: testDataDir},
-	)
+	sc := &CSV{CSVVersion: csvVer, pathPrefix: testDataDir}
+	err := s.Execute(&input.Config{ProjectName: projectName}, sc)
 	if err != nil {
 		t.Fatalf("Failed to execute the scaffold: (%v)", err)
 	}
 
+	csvStr := buf.String()
+	tsSplit := regexp.MustCompile("creationTimestamp: \".+\"").Split(csvStr, 2)
+	if len(tsSplit) == 2 {
+		csvStr = tsSplit[0] + `creationTimestamp: "2019-01-01T00:00:00Z"` + tsSplit[1]
+	} else {
+		t.Errorf("Expected to find creationTimestamp in CSV metadata, found none")
+	}
```

Comment:
Just making sure I understand. We're substituting the timestamp here since executing the scaffolds uses the current time, and we don't want the test to fail because of this one difference?

It might be nice to extract this into a separate function so it's more obvious what we're doing and why (and also for reuse if we ever need it in other tests).

Reply:
Yeah exactly. I'll create a separate function.